compose/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Mon Nov 14 11:27:20 CST 2022
+gradle.version=7.4
```

Comment:
Looks like the compose project gitignore is not done properly and therefore we have the `.gradle` folder here.
I will fix that.